docs/container.md

```diff
@@ -68,7 +68,7 @@ class Config extends ContainerConfig
         $adr->middle('My\Middleware\BarHandler');
         $adr->middle('Radar\Adr\Handler\RoutingHandler');
         $adr->middle('Radar\Adr\Handler\ActionHandler');
-        $adr->middle('Radar\Adr\Handler\SendingHandler');
+        $adr->middle('Radar\Adr\Responder\Responder');
```

Comment:
Yes, I think ResponseSender. Believe it or not, ResponseSender might be best as the first entry, rather than the last, so it can send any responses that get generated by other middlewares.